### PR TITLE
Template changes

### PR DESCRIPTION
**Pull request by aconsilvio**

### CODE REVIEW

**Comment by aconsilvio** [comment]
Fixes #70

File: sass/stylesheets/signin.sass [hunk 1]
```diff
@@ -28,3 +28,73 @@ body
       margin-bottom: 10px
       border-top-left-radius: 0
       border-top-right-radius: 0
+
```
Comment (davidabrahams) [inline sass/stylesheets/signin.sass]: isn't all this the same as the sass code that's in `main`?

Comment (aconsilvio) [inline sass/stylesheets/signin.sass]: Yes but main never gets called for the signin page - we can change that if you want, but the last i remember we were trying to keep different templates separate?  correct me if i'm wrong. :)  But yes, all this button code is exactly the same

**Comment by aconsilvio** [comment]
Also, can everyone test this - sometimes when i click on a bunch of student the header dropdowns stop working.  I want to make sure this is a consistent problem before I try figuring out what it is because as of now I don't see any reason for this to be happening/sometimes my internet is terrible.

File: sass/stylesheets/signin.sass (hunk 1 above)
Comment (davidabrahams) [inline sass/stylesheets/signin.sass]: Nope you're right. I guess maybe if we're copying over a large portion of the `main` css into the `login` css, at that point we might as well just reference `main` css on the login page. Thoughts?

**Comment by aconsilvio** [comment]
Also can people check on the sliders - they seem to be acting weird too

**Comment by nmohamed** [comment]
thoughts:
-remove the border around buttons? they look kinda weird (the login buttons)
-when i choose a date i get "date is undefined error"
-i think it'd make more sense for the student page to have the tabs and blue box take the whole page, and put the student name alongside the tabs. then, show student information can just be another tab. the "student info goes here" part isn't really necessary anymore since it'd be a tab
-action items could be another tab then (which is the default view then?) or just apart of the view data portion i'm working on
-with the above case, the entire screen would be blue with the current colors so that'd probably mean switching the box thing to be white and tabs to be blue instead
-also, the blue background doesn't work well with some of the elements. we'd have to change colors of the sliders
-my preferences on color scheme: i'm not a big fan of the current colors. personally i'd prefer the sidebar to be either really light grey or a dark navy/other color, and the general colors to be a bit more saturated. anyway we should all talk about this tonight
-search bar input box should have rounded corners like the rest of the inputs
-also, cool effects to possibly add: hovering over students can animate the div to slowly fade in/out the new color

**Comment by BrennaManning** [comment]
:+1: I like it! Lots of blue on the forms but looks nice. Functionality is good except for strange slider problem so far only seen on Annabel's computer, and we should change text color for engaging values so they are visible.

**Comment by davidabrahams** [comment]
So it turns out we never included `boostrap.js`, and the commit where we included `dropdown.js` we also included `bootstrap.js`, which has `dropdown.js` in it...we should only do one of them, probably `bootstrap.js` since it makes it so we never have this problem again. thoughts?

**Comment by davidabrahams** [comment]
Alright. I can't seem to figure out why dropdown.js is required, but there seem to be bugs popping up whenever it's not included. I'll leave it as a personal learning task to try to figure out why it's necessary, and for now leave it.

**Comment by davidabrahams** [comment]
MERGE THE BUS :bus:

**Pull request merged**